### PR TITLE
Changelog for 2.0

### PR DESCRIPTION
**Pull request by Nyholm**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,18 @@
...
+
+- Support from HTTPlug 2.0 and PSR-18
+
+### Changed
+
+- `Client` and `Promise` are both final. 
+
+## Removed
+
+- Support for PHP <7.1
```
Comment (sagikazarmark) [inline CHANGELOG.md]: This should go into 1.2 version IMHO.

Comment (Nyholm) [reply CHANGELOG.md]: How come?

Comment (sagikazarmark) [reply CHANGELOG.md]: Well, not sure. Looking at the diff between the latest release and the current master there are quite a few changes. But as I took a closer look, nothing special, so we might not have to release 1.2 after all. I had the impression from the number of commits that we should probably release a last 1.x version before releasing 2.0

**Comment by Nyholm** [comment]
:+1: for merging

**Pull request merged**